app/src/test/java/com/cesar/knot_thing_lib_android/ExampleUnitTest.kt

```diff
@@ -0,0 +1,17 @@
+package com.cesar.knot_thing_lib_android
```

Comment:
please remove this class if you are not testing anything

Reply:
Done.